docs/linting/Troubleshooting.mdx

```diff
@@ -30,21 +30,67 @@ If you don't find an existing extension rule, or the extension rule doesn't work
 
 ## I get errors telling me "ESLint was configured to run ... However, that TSConfig does not / none of those TSConfigs include this file"
 
+These errors come are caused by an ESLint config requesting type information be generated for a file that isn't included in the TypeScript configuration.
+
 ### Fixing the Error
 
+<!-- prettier-ignore-start -->
```

Comment:
Am I doing this wrong? Try removing these disables - is this a bug in Prettier / its mdx parsing? Or is it just me?